### DISCUSSION TITLE
VESA to text mode

### DISCUSSION THREAD

**Discussion by dyter69** [PeterSuh-Q3/tinycore-redpill#26] [Q&A] [answered]
Hello, I have a very small screen, can you and an option in boot menu for VESA to text mode ?

**Comment by PeterSuh-Q3** [comment, marked as answer]
Try adding "set vesa_mode=1" just below the set timeout_style="menu" line, as in the example of rr's grub.cfg below.

While the Friend kernel is booting, stop booting with ctrl+c and
You can edit grub.cfg with the command below.
This part is permanently saved.

vi /mnt/tcrp-p1/boot/grub/grub.cfg

https://xpenology.com/forum/topic/61839-tinycore-redpill-loader-build-support-tool-m-shell/?do=findComment&comment=454535
